src/Raven.Server/Documents/Handlers/StatsHandler.cs

```diff
@@ -110,5 +112,22 @@ public Task BytesMetrics()
 
             return Task.CompletedTask;
         }
+
+        [RavenAction("/databases/*/periodic-export-stats", "GET")]
```

Comment:
maybe /status?

The JSON here is:
{
    Status: { ... }
}

Shouldn't it be just the status without additional wrapping in Status property?